supplier/views.py

```diff
@@ -0,0 +1,32 @@
+from django.shortcuts import render
+from supplier.models import Supplier
+from .forms import SupplierForm
+
+
+def signup_supplier(request):
+    form = SupplierForm()
+    context = {'form': form, 'message': ['username should be longer than 6',
+                                         'password should be longer than 6',
+                                         'business name should be longer than 6']}
+    if request.method == 'POST':
+        form = SupplierForm(request.POST)
+        user_name = request.POST.get('user_name')
+        password = request.POST.get('password')
```

Comment:
a Password **MUST** be encrypted. never enter a password as plain text to the DB.

Reply:
I know at the moment i was testing that all done and works